docs/cli-auth-tokens.md

```diff
@@ -0,0 +1,75 @@
+# Using Authentication Tokens with Celestia Node CLI
```

Comment:
why not make a PR to update the celestia docs: https://github.com/celestiaorg/docs/blob/main/tutorials/node-tutorial.md#auth-token--auth-token
instead of trying to add a new page here?